packages/components/src/menu/styles.ts

```diff
@@ -380,7 +380,7 @@ export const MenuItemHelpText = styled( Truncate )`
 	font-size: ${ font( 'helpText.fontSize' ) };
 	line-height: 16px;
 	color: ${ LIGHTER_TEXT_COLOR };
-	word-break: break-all;
+	word-break: break-word;
```

Comment:
`word-break: break-word` is [deprecated](https://developer.mozilla.org/en-US/docs/Web/CSS/word-break#browser_compatibility), so we should use the non-deprecated equivalent:

```suggestion
	overflow-wrap: anywhere;
```

Reply:
Hey @mirka, Thanks for the suggestion. 

I have made the suggested changes. Please let me know if there is anything else that can be done.